### DISCUSSION TITLE
luci-app-***包在这个项目是怎么实现自动拉取最新版本的呢？

### DISCUSSION THREAD

**Discussion by GoFightNow** [immortalwrt/immortalwrt#1576] [General]
我本来用的是openwrt，最近使用值守更新，发现原版仓库没有openclash等包，报错无法升级。
看了immortalwrt，luci包不是bot自动提交的，都是tianling大佬的提交。
我想简单了解下这个项目是怎么实现监控这么多包，并且都自动提交新版本到仓库的。
有没有大佬能解惑的

**Comment by 1715173329** [comment]
bio robot, men.
